src/utils/MasterFisher.ts

```diff
@@ -79,6 +79,12 @@ export class MasterFisher implements Fishable {
     const history = [meta];
     let [sdType, parent] = [meta.sdType, meta.parent];
     while (sdType == null && parent != null) {
+      // A logical model with the Base parent will not have a sdType value. In this case, we know
+      // that the sdType should be the logical model's url and not be based on its parent.
+      if (parent === 'Base') {
+        [sdType, parent] = [meta.url, parent];
+        continue;
+      }
```

Comment:
I think this is too specific.  We want to do this for *every* logical model (whether it derives from `Base` or not).  If it is a logical model, then its type should be its own URL, right?  As it is now, I'm still getting the wrong type when I do this:
```
Logical: FooFromElement
Parent: Element
* bars 0..* BarFromElement "The bars of the foo"

Logical: BarFromElement
Parent: Element
* height 1..1 Quantity "The height of the bar"
```
Result:
```
"type": [
  {
    "code": "Element",
    "profile": [
      "http://logicals.org/StructureDefinition/BarFromElement"
    ]
  }
]
```

Or this:
```
Logical: Other
* thing 1..1 boolean "Is it a thing?"

Logical: FooFromOther
Parent: Other
* bars 0..* BarFromOther "The bars of the foo"

Logical: BarFromOther
Parent: Other
* height 1..1 Quantity "The height of the bar"
```
Result:
```
"type": [
  {
    "code": "http://logicals.org/StructureDefinition/Other",
    "profile": [
      "http://logicals.org/StructureDefinition/BarFromOther"
    ]
  }
]
```